analysis/run_supercloud_experiments.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 START_SEED=456
```

Comment:
@tomsilver will revert changes to this file

Reply:
don't forget!

Reply:
lets see if we want to relaunch supercloud after whatever changes you make